pontoon/base/views.py

```diff
@@ -314,11 +317,7 @@ def contributor(request, email, template='user.html'):
         raise Http404
 
     # Exclude unchanged translations
-    translations = (
-        Translation.objects.filter(user=user)
-        .exclude(string=F('entity__string'))
-        .exclude(string=F('entity__string_plural'))
-    )
+    translations = User.translations.get_changed_translations().filter(user=user)
```

Comment:
`User.translations` doesn't exist, and it returns a list of users anyway, whereas `translations` here needs to be a list of translations.

I think you should remove your changes to the `contributor` view and focus on the leaderboard for this PR. It's true that we eventually want to remove these excessive DB queries for the contributor view as well, but we should do that in a PR after this one to keep this one simple.